elasticdownload/src/main/java/is/arontibo/library/ElasticDownloadView.java

```diff
@@ -107,5 +107,33 @@ public void onEnterAnimationFinished() {
 
         // Do further actions if necessary
     }
+    
+    
+    /////////////////////////////////////////////////////////////////////////////
+    ///Added on 2016-07-29 by PGMacDesign as per Issue #27 in Github/////////////
+    /////////////////////////////////////////////////////////////////////////////
+    /**
+     * Set the background color of the Elastic Download View
+     * @param passedColor int Color Color to set the background
+     */
+    public void setBackgroundColor(int passedColor){
```

Comment:
Take a look at how the rest of the file is formatted (I'm talking about braces here), there's always one space between one parenthesis and any curly brace. Try to keep some consistency on the code styling :)